### DISCUSSION TITLE
default save path of icub app sdf file

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#645] [Q&A] [answered]
Hi, does the default save path of icub app sdf file like `tutorial_cartesian-interface.sdf` locates in the `tutorial_cartesian-interface/gazebo/worlds` ？
I found one in the path, but not sure if or not.

**Comment by jimingda** [comment, marked as answer]
I think i found the correct path, inside the docker: `/usr/local/share/gazebo/worlds/tutorial_cartesian-interface.sdf`
And in the ubuntu terminal, the path is `/exports/code/robotology-superbuild/build/install/share/gazebo/worlds/tutorial_cartesian-interface.sdf`

**Reply by traversaro** [reply]
The important thing is that the folder in which the sdf is found is in a directory contained in the `GAZEBO_RESOURCE_PATH` environment variable.
